### PR TITLE
Rspamd Jail to protect the Web UI

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/5663

### CODE REVIEW

**Comment by gsanchietti** [comment]
Even if the implementation should work, I don't think it's really a good idea relying on httpd-admin log.
Is there any chance to grab the same information from rspamd log?

**Comment by stephdl** [comment]
Quite zero chance because we do a reverse proxy from httpd admin to rspamd tcp port and the UI of rspamd is javascript (Jquery) with no traces in the rspamd logs. Moreover all authentications comes from apache on 980

File: root/etc/fail2ban/filter.d/rspamd.conf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[Definition]
+#this filter is made against brute force attack to rspamd
+# Author Stephane de Labrusse <stephdl@de-labrusse.fr>
+
+failregex = ^<HOST> - (-|\w+) \[*\] "GET /rspamd/ HTTP/1.1" 401 \d+$
```
Comment (gsanchietti) [inline root/etc/fail2ban/filter.d/rspamd.conf]: After @DavidePrincipi suggestion, I'd change the regexp to match also all rspamd sub-paths like "rspamd/stat", "/rspamd/img/" and so on.

File: root/etc/fail2ban/filter.d/rspamd.conf [hunk 2]
```diff
@@ -2,6 +2,6 @@
 #this filter is made against brute force attack to rspamd
 # Author Stephane de Labrusse <stephdl@de-labrusse.fr>
 
-failregex = ^<HOST> - (-|\w+) \[*\] "GET /rspamd/ HTTP/1.1" 401 \d+$
+failregex = ^<HOST> - (-|\w+) \[*\] "GET /rspamd/* HTTP/1.1" 401 \d+$
```
Comment (DavidePrincipi) [inline root/etc/fail2ban/filter.d/rspamd.conf]: The regex is not correct, but matches anyway :smile: 

Should be `/rspamd/.*`

**Pull request merged**